strongbox-storage/strongbox-storage-indexing/pom.xml

```diff
@@ -244,9 +244,21 @@
         </dependency>
 
         <dependency>
+            <groupId>org.carlspring.maven</groupId>
+            <artifactId>artifact-downloader</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
Do we need this dependency? Isn't it coming as a dependency of the `artifact-downloader`? (Just asking)...